Olympus2.hebontes/CfgFunctions.hpp

```diff
@@ -9,6 +9,7 @@ class CfgFunctions {
 			class range40mmTargetGroups;
 			class rangeAdvMarksmanGroups;
 			class rangeBasicLMGTargetGroups;
+            class rangeDrivingCourseGroups;
```

Comment:
```suggestion
            class courseBasicDrivingTargetGroups;
```
It's more of a course than a range.
Also add `Basic`, mostly in case we ever do an advanced one (doubtful at this moment, but may as well have it future proofed)